File: delta/src/main/java/com/delta/fly/dto/SeatDTO.java
package com.delta.fly.dto;

import com.delta.fly.enumeration.Class;

public class SeatDTO {

    private Integer rowNo;
    private Integer colNo;
    private Class seatClass;

    public Integer getRowNo() {
        return rowNo;
    }

    public void setRowNo(Integer rowNo) {
        this.rowNo = rowNo;
    }

    public Integer getColNo() {
        return colNo;
    }

    public void setColNo(Integer colNo) {
        this.colNo = colNo;
    }

    public Class getSeatClass() {
        return seatClass;
    }

    public void setSeatClass(Class seatClass) {
        this.seatClass = seatClass;
    }
}

File: delta-client/src/app/modules/moderation/airplane/seat-dto.ts
import {SeatClass} from '../../shared/enumeration/seat-class.enum';

export class SeatDTO {
  rowNo: number;
  colNo: number;
  seatClass: SeatClass;
  constructor(rowNo?: number, colNo?: number, seatClass?: SeatClass) {
    this.rowNo = rowNo;
    this.colNo = colNo;
    this.seatClass = seatClass;
  }
}
